src/test/groovy/hudson/plugins/gradle/injection/BuildScanInjectionGradleWithDurableTaskStepUseWatchingIntegrationTest.groovy

```diff
@@ -75,6 +75,11 @@ class BuildScanInjectionGradleWithDurableTaskStepUseWatchingIntegrationTest exte
         then:
         j.assertLogContains('password=****', secondRun)
         j.assertLogNotContains(secret, secondRun)
+
+        cleanup:
+        System.err.println('---%<--- agent logs')
+        agent.computer.logText.writeLogTo(0, System.err)
+        System.err.println('--->%---')
```

Comment:
Otherwise you would not notice the errors. Currently `JenkinsRule.create[Online]Slave` does not stream logs. (`InboundAgentRule` does.)

Reply:
ooh I see, I wondered as well why I could not see the error